### PR TITLE
Add more documentation for server setup and database setup/seed/restore

### PR DESCRIPTION
**Pull request by blushi**
closes: regen-network/regen-registry#666

### CODE REVIEW

**Comment by blushi** [comment]
Still some TODOs but it'd be good if you could have a first look at this PR to make sure I cover all the important points @mhagel @haveanicedavid

File: sql/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+```
+Finally, just run the `DROP FUNCTION` statement on the function that needs to be deleted.
+
+### Versioned migrations
+
+For versioned migrations, we use a streamlined version of SemVer, with only major and minor numbers, this is being used when naming migration files, e.g. `V1_10`. So far, we've only been incrementing the minor number but we might consider incrementing the major number too in case of major release.
+
+*Dev tip*:
+
+When adding complex row level security policies, it's usually best to add them manually and test them before dropping them and writing down their final version in a new migration, mainly for the reason exposed in the next section.
```
Comment (haveanicedavid) [inline sql/README.md:33]: 👌

**Review by haveanicedavid — APPROVED** [review]
Looks great - should be really helpful for onboarding moving forward!

**Review by mhagel — APPROVED** [review]
Thank you - this will be valuable

**Pull request merged**